L1Trigger/L1THGCal/interface/HGCalTriggerFECodecBase.h

```diff
@@ -99,7 +101,7 @@ namespace HGCalTriggerFE {
           << "Data payload was already set for HGCTriggerFECodec: "
           << this->name() << " overwriting current data!";
       }
-      static_cast<Impl&>(*this).setDataPayloadImpl(mod,ee,fh,bh);
+      static_cast<Impl&>(*this).setDataPayloadImpl(mod,geom,ee,fh,bh);
```

Comment:
The module class gives the list of trigger cell detids (https://github.com/PFCal-dev/cmssw/blob/hgc_trigger_fwk_SLHC26p3/L1Trigger/L1THGCal/interface/HGCalTriggerGeometryBase.h#L99), not the TriggerCell objects. To get the HGC cells inside each trigger cell we need the geometry (or the TriggerCell objects), right?